File: VB/Contacts/Reports/PersonReport.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Drawing
Imports System.Collections
Imports System.ComponentModel
Imports DevExpress.XtraReports.UI
Imports DevExpress.Xpo
Imports Contacts.Data

Namespace Contacts.Reports
	Partial Public Class PersonReport
		Inherits XtraReport
		Implements IReport
		Public Sub New()
			InitializeComponent()
		End Sub


		Public Sub LoadDataSource() Implements IReport.LoadDataSource
			DataSource = New XPCollection(Of Person)()
		End Sub
	End Class
End Namespace

File: CS/Contacts/Reports/PersonReport.cs
using System;
using System.Drawing;
using System.Collections;
using System.ComponentModel;
using DevExpress.XtraReports.UI;
using DevExpress.Xpo;
using Contacts.Data;

namespace Contacts.Reports
{
    public partial class PersonReport : XtraReport, IReport
    {
        public PersonReport()
        {
            InitializeComponent();
        }


        public void LoadDataSource()
        {
            DataSource = new XPCollection<Person>();
        }
    }
}
